yjit/src/stats.rs

```diff
@@ -553,6 +553,7 @@ make_counters! {
     block_next_count,
     defer_count,
     defer_empty_count,
+    deleted_defer_block_count,
```

Comment:
WDYT about adding this to `DEFAULT_COUNTERS`? I want to monitor the value whether `--yjit-stats` is given or not.